docs/en/users/07_Frequently_Asked_Questions.md

```diff
@@ -24,14 +24,14 @@ The feed syntax can be invalid, it can be unrecognized by the SimplePie library.
 The first step is to identify what causes the problem.
 Here are the steps to follow:
 
-1. __Verify if the feed syntax is valid__ with the [W3C on-line tool](http://validator.w3.org/feed/ "RSS and Atom feed validator"). If it is not valid, there is nothing we can do.
-1. __Verify SimplePie validation__ with the [SimplePie on-line tool](http://simplepie.org/demo/ "SimplePie official demo"). If it is not recognized, there is nothing we can do.
-1. __Verify FreshRSS integration__ with the [demo](http://demo.freshrss.org "FreshRSS official demo"). If it is not working, you need to [create an issue on Github](https://github.com/FreshRSS/FreshRSS/issues/new "Create an issue for FreshRSS") so we can have a look at it. If it is working, there is probably something fishy with the hosting server.
+1. __Verify if the feed syntax is valid__ with the [W3C on-line tool](https://validator.w3.org/feed/ "RSS and Atom feed validator"). If it is not valid, there is nothing we can do.
+1. __Verify SimplePie validation__ with the [SimplePie on-line tool](https://simplepie.org/demo/ "SimplePie official demo"). If it is not recognized, there is nothing we can do.
+1. __Verify FreshRSS integration__ with the [demo](https://demo.freshrss.org "FreshRSS official demo"). If it is not working, you need to [create an issue on Github](https://github.com/FreshRSS/FreshRSS/issues/new "Create an issue for FreshRSS") so we can have a look at it. If it is working, there is probably something fishy with the hosting server.
 
 Here is a list of feeds which don't work:
 
-* http://foulab.org/fr/rss/Foulab_News: is not a W3C valid feed (November 2014)
-* http://eu.battle.net/hearthstone/fr/feed/news: is not a W3C valid feed (Novembre 2014)
+* https://foulab.org/fr/rss/Foulab_News: is not a W3C valid feed (November 2014)
```

Comment:
Not related to HTTPS, but that's a 404 now.

The new URL seems to be https://foulab.org/news/index.xml which looks valid enough at a glance.

Reply:
I understand the example was provided for showing a non-working feed and I just try to see if the http and https version were the same.
I did not check the validation process just https vs http.